pkg/utils/gateway_wrapper.go

```diff
@@ -4,12 +4,11 @@ import (
 	"fmt"
 	"strings"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
wondering if this really belongs in utils or perhaps it could be also added to the pkg/apis ?